diff -uraN SDL2-2.0.22/CMakeLists.txt omv-SDL2-2.0.22/CMakeLists.txt
--- SDL2-2.0.22/CMakeLists.txt	2022-04-21 19:37:37.000000000 +0200
+++ omv-SDL2-2.0.22/CMakeLists.txt	2022-05-16 16:26:02.267814221 +0200
@@ -1382,7 +1382,7 @@
         # We've already found the header, so REQUIRE the lib to be present
         pkg_search_module(UNWIND REQUIRED libunwind)
         pkg_search_module(UNWIND_GENERIC libunwind-generic)
-        list(APPEND EXTRA_LIBS ${UNWIND_LIBRARIES} ${UNWIND_GENERIC_LIBRARIES})
+        list(APPEND EXTRA_LIBS ${UNWIND_LIBRARIES} -L${UNWIND_GENERIC_LIBDIR} -Wl,-rpath,${UNWIND_GENERIC_LIBDIR} ${UNWIND_GENERIC_LIBRARIES})
       endif()
     endif()
 
