--- a/CMakeLists.txt	2013-08-12 06:57:53.000000000 +0400
+++ b/CMakeLists.txt	2013-09-12 15:08:44.000000000 +0400
@@ -143,9 +143,9 @@
   set(CMAKE_C_FLAGS "$ENV{CFLAGS}")
   list(APPEND EXTRA_CFLAGS "$ENV{CFLAGS}")
 endif("$ENV{CFLAGS}" STREQUAL "")
-if(NOT ("$ENV{CFLAGS}" STREQUAL "")) # Hackish, but does the trick on Win32
-  list(APPEND EXTRA_LDFLAGS "$ENV{LDFLAGS}")
-endif(NOT ("$ENV{CFLAGS}" STREQUAL ""))
+#if(NOT ("$ENV{CFLAGS}" STREQUAL "")) # Hackish, but does the trick on Win32
+#  list(APPEND EXTRA_LDFLAGS "$ENV{LDFLAGS}")
+#endif(NOT ("$ENV{CFLAGS}" STREQUAL ""))
 
 # Those are used for pkg-config and friends, so that the SDL2.pc, sdl2-config,
 # etc. are created correctly.
