--- SDL2-2.0.3/CMakeLists.txt.0001~	2014-10-16 19:45:02.187347638 +0200
+++ SDL2-2.0.3/CMakeLists.txt	2014-10-16 19:45:38.448019788 +0200
@@ -573,6 +573,7 @@ foreach(_SUB ${SDL_SUBSYSTEMS})
   endif()
 endforeach()
 if(SDL_JOYSTICK)
+  file(GLOB JOYSTICK_SOURCES ${SDL2_SOURCE_DIR}/src/joystick/*.c ${SDL2_SOURCE_DIR}/src/core/linux/SDL_udev.c)
   file(GLOB JOYSTICK_SOURCES ${SDL2_SOURCE_DIR}/src/joystick/*.c)
   set(SOURCE_FILES ${SOURCE_FILES} ${JOYSTICK_SOURCES})
 endif()
