diff -uraN SDL2-2.30.5/CMakeLists.txt omv-SDL2-2.30.5/CMakeLists.txt
--- SDL2-2.30.5/CMakeLists.txt	2024-07-03 23:23:37.000000000 +0200
+++ omv-SDL2-2.30.5/CMakeLists.txt	2024-07-04 20:33:50.965228068 +0200
@@ -1676,7 +1676,7 @@
     endif()
     if((LINUX OR FREEBSD) AND HAVE_LINUX_INPUT_H AND NOT ANDROID)
       set(SDL_JOYSTICK_LINUX 1)
-      file(GLOB JOYSTICK_SOURCES ${SDL2_SOURCE_DIR}/src/joystick/linux/*.c ${SDL2_SOURCE_DIR}/src/joystick/steam/*.c)
+      file(GLOB JOYSTICK_SOURCES ${SDL2_SOURCE_DIR}/src/joystick/linux/*.c ${SDL2_SOURCE_DIR}/src/joystick/steam/*.c ${SDL2_SOURCE_DIR}/src/core/linux/SDL_udev.c)
       list(APPEND SOURCE_FILES ${JOYSTICK_SOURCES})
       set(HAVE_SDL_JOYSTICK TRUE)
     endif()
